rdserial/dps/tool.py

```diff
@@ -276,6 +312,74 @@ def loop(self):
             else:
                 return
 
+    def guiloop(self):
+        while True:
+            try:
+                device_state = self.assemble_device_state()
```

Comment:
I find my 6006 is crashing from time to time, I think it's maybe being polled too rapidly - I might put a delay here to make it only update once a second or something.